src/vi/ui/viui.py

```diff
@@ -299,7 +301,7 @@ def readAndApplySettings(self):
         self.restoreState(qsettings.value("saveState", self.saveState()))
         self.move(qsettings.value("pos", self.pos()))
         self.resize(qsettings.value("size", self.size()))
-        if qsettings.value("maximized", self.isMaximized()):
+        if qsettings.value("maximized", self.isMaximized()) == "true":
```

Comment:
QSettings values are apparently always strings.  The others here might need updates as well - I only fixed what was breaking on load for me.